_data/utilities.yml

```diff
@@ -450,7 +450,7 @@ websites:
       tfa: No
 
     - name: Vodafone (IT)
-      url: http://www.vodafone.it/
+      url: https://www.vodafone.it/
```

Comment:
I can't load the site over HTTPS. Is it just me?
Maybe they don't support modern ciphers or something. 🤔

Reply:
Firefox, doing TLS 1.2, redirects to https://www.vodafone.it/portal/Privati immediately when entering the https URL, I wouldn't know where they get the information from, that it's a private account I'd want to open, but https works for me.

Reply:
Found the issue:

![chrome_2019-05-07_21-19-23](https://user-images.githubusercontent.com/3535780/57326900-ced3a080-710d-11e9-8d62-9d6d2b559b1a.png)
![firefox_2019-05-07_21-20-13](https://user-images.githubusercontent.com/3535780/57326937-ea3eab80-710d-11e9-9bc1-3c3e6a958db3.png)
![firefox_2019-05-07_21-19-48](https://user-images.githubusercontent.com/3535780/57326920-ddba5300-710d-11e9-91c1-c69090d4df2e.png)

Maybe we should continue with the HTTP version for now? I'm guessing more users than me are going to have problems loading the site over HTTPS.

Reply:
Ah, ffs, Vodafone ...